src/test/scala/io/iohk/ethereum/blockchain/sync/FastSyncSpec.scala

```diff
@@ -26,6 +26,7 @@ class FastSyncSpec
 
   class Fixture extends EphemBlockchainTestSetup with TestSyncConfig with TestSyncPeers {
     override implicit lazy val system = FastSyncSpec.this.system
+    override implicit val scheduler = FastSyncSpec.this.scheduler
```

Comment:
WDYT about this:
```scala
class FastSyncSpec
    extends TestKit(ActorSystem("FastSync_testing"))
    // ...
    with WithActorSystemShutDown { self =>
    ...    
    override implicit val scheduler = self.scheduler
    ```
    I have seen `OuterClass.this.sth` in OOP world, but in Scala people use mainly ☝️  approach.